### PR TITLE
PCG solver + shape factor + corrections

### PR DESCRIPTION
**Pull request by puigmontella**
Pull request to:

-  replace GAMG algebraic solvers in  1D tutorials by PCG solver. 
- include the contribution of the shape factor in interfacial models. Shape factor adds the possible irregularities of particles. Spherical grains have a shape factor of 1, while very irregular particles have a shape factor of 0.3.  
- small corrections in Peqn and Ufall.

### CODE REVIEW

File: solver/pU/pEqn.H [hunk 1]
```diff
@@ -199,9 +199,9 @@ License
             (
                 phiHbyA.boundaryField()
               - alphaf.boundaryField()
-               *(mesh.Sf().boundaryField() & Ua.boundaryField())
+               *((mesh.Sf().boundaryField() & Ua.boundaryField())
```
Comment (CyrilleBonamy) [inline solver/pU/pEqn.H:202]: Are you sure?
I think there's a bug with the brackets...
Can you check?

**Review by CyrilleBonamy — COMMENTED** [review]
Thank you very much for your contribution.

Can you check my comments, and maybe add a few words in your PR comment about the dragModel changes (role of shape factor) ?

File: solver/pU/pEqn.H (hunk 1 above)
Comment (puigmontella) [reply solver/pU/pEqn.H]: re-checked and re-corrected!

File: solver/pUf/pEqn.H [hunk 2]
```diff
@@ -236,10 +236,10 @@ License
             p_rbgh.boundaryFieldRef(),
             (
                 phiHbyA.boundaryField()
-              - alphaf.boundaryField()
+              - (alphaf.boundaryField()
```
[the remark this answers is not in the document]
Comment (puigmontella) [reply solver/pUf/pEqn.H:237]: re-checked and re-corrected!

**Pull request merged**